x-pack/plugins/actions/server/cleanup_failed_executions/lib/space_id_to_namespace.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SpacesPluginStart } from '../../../../spaces/server';
+
+export function spaceIdToNamespace(spaces?: SpacesPluginStart, spaceId?: string) {
+  return spaces && spaceId ? spaces.spacesService.spaceIdToNamespace(spaceId) : undefined;
```

Comment:
heh, bet ya $5 we have at least one other version of this function in actions, somewhere :-)

Reply:
Fixed in https://github.com/elastic/kibana/pull/96971/commits/8fda03e25a8b4064ba3937dad1850bf698cce8b7 :-)